content/en/quality_gates/setup/_index.md

```diff
@@ -0,0 +1,188 @@
+2. Invoke Quality Gates by using the `datadog-ci gate evaluate` command in your CI pipeline.
+
+## Create a rule
+
+To create Quality Gates rules for your organization, your user account must have the `quality_gate_rules_write` [permission][1].
+
+1. In Datadog, navigate to [**CI** > **Quality Gate Rules**][2] and click **+ New Rule**.
+2. Define the rule scope. The rule scope defines when the rule should be evaluated. For example, you can create a rule that it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to `select when to evaluate` and add included or excluded values for the scope. See [rule scopes](#rule-scope) for more information.
+3. Select the rule type. You can choose between `Static Analysis` and `Test`.
+4. Define the rule condition. The rule condition states in which scenario the rule fails, failing the related pipeline (if the rule is blocking). You can select one of the existing rule conditions for the rule type you have selected.
+
+The following example shows how to create a static analysis rule that fails when there are one or more static analysis violations with "error" status and "security" category being introduced in a specific commit:
+
+   {{< img src="ci/qg_rule_condition_sa_errors_security_2.png" alt="Rule for static analysis security errors" style="width:80%;">}}
+
+5. Select whether the rule should block the pipeline or not when it fails.
+   Non-blocking rules are helpful when rolling out a new rule and wanting to verify its behavior before making it blocking.
+6. Select the time window over which the query is evaluated.
+7. Select a rule name that describes the rule that you are creating.
+8. Click **Save Rule**.
+
+
+## Invoking quality gates
+
+To use quality gates, your [`datadog-ci`][7] version should be `2.19.0` or later.
+
+You can invoke the Quality Gates evaluation by calling the [`datadog-ci gate evaluate`][4] command.
+
+This command:
+
+1. Retrieves all the rules that have [rule scopes](#rule-scope) and [custom scopes](#custom-scope) matching the current pipeline context (repository, branch, or custom scope passed in the command).
+2. Evaluates all the matching rules.
+3. If one or more blocking rules fail, the command fails as well, blocking the pipeline.
+
+<div class="alert alert-danger">For the command to work properly, it's important that the events (tests, static analysis violations)
+are sent to Datadog <strong>before</strong> the <code>datadog-ci gate evaluate</code> command is executed.
+Otherwise, the rules might have an incorrect behavior due to the absence of the events.
+</div>
+
+The command requires the `DD_API_KEY` and `DD_APP_KEY` environment variables to point to your [Datadog API Key][5]
+and [Datadog Application Key][6]. Also, you need to set the `DD_BETA_COMMANDS_ENABLED` environment
+variable as `true`. Optionally, you can specify the `DD_SITE` environment variable to point to a specific datadog site (default value is `datadoghq.com`).
```

Comment:
Yeah, I think that makes sense to make it a list. But I think it should stay where it is, since it's about how to invoke the command properly.

These are all things you add to the CLI command, which we have a code component for, shown below this line.

Reply:
```suggestion
| Environment Variables | Description |
|---|---|
| `DD_API_KEY` | Point to your [Datadog API key][5]. |
| `DD_APP_KEY` | Point to your [Datadog application key][6]. |
| `DD_BETA_COMMANDS_ENABLED` | Set to `true`. |
| `DD_SITE` | (Optional) Point to a specific [Datadog site][12] (default value is `datadoghq.com`). |
| LOGGING_MAXDAYS | Number of days to keep file logs on the system before deleting them. Can be any number when running an unattended installation. |
| WORKERCONFIG_FILEPATH | This should be changed to the path to your Synthetics Private Location Worker JSON configuration file. Wrap this path in quotes if your path contains spaces. |
```